File: rest/taskrouter/workers/instance/post/example-1/example-1.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest import TwilioTaskRouterClient

# Your Account Sid and Auth Token from twilio.com/user/account
account_sid = "{{ account_sid }}"
auth_token  = "{{ auth_token }}"
workspace_sid = "{{ workspace_sid }}"
worker_sid = "{{ worker_sid }}"

client = TwilioTaskRouterClient(account_sid, auth_token)

activity = client.workers(workspace_sid).update(worker_sid, attributes='{"type":"support"}')
print activity.friendly_name

# alternatively
worker = client.workers(workspace_sid).get(worker_sid)
worker = worker.update(attributes='{"type":"support"}')
print worker.attributes

File: rest/taskrouter/activities/instance/post/example-1/example-1.php
<?php
// Get the PHP helper library from twilio.com/docs/php/install
require_once('/path/to/twilio-php/Services/Twilio.php'); // Loads the library

// Your Account Sid and Auth Token from twilio.com/user/account
$accountSid = "{{ account_sid }}";
$authToken = "{{ auth_token }}";
$workspaceSid = "{{ workspace_sid }}";
$activitySid = "{{ activity_sid }}";

$client = new TaskRouter_Services_Twilio($accountSid, $authToken, $workspaceSid);

$activity = $client->workspace->activities->get($activitySid);
$activity->update(
	array(
		'FriendlyName' => 'NewAvailableFriendlyName',
		'Available' => 'true'
	)
);
